code/index.html

```diff
@@ -1,30 +1,156 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
+    <link rel="stylesheet" href="./style.css ">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@500&family=Parisienne&display=swap" rel="stylesheet">
+    <title>Buttercream Piping Masterclass</title>
     <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
+  
+    
     <!-- video or image as a header is cool :) -->
+    <header class="header">
+      <picture>
+        <source srcset="./assets/cupcakes-mobile.jpg" media="(max-width: 767px)"> <!--mobile-->
+        <source srcset="./assets/cupcakes-tablet.jpg" media="(max-width: 1199px)"><!--tablet-->
+        <img class="hero-image" src="./assets/cupcakes-desktop.jpg" alt="Cupcake hero image"/><!--desktop-->
+      </picture>
 
+      <div class="hero-text">
+        <h1 class="hero-title">
+          Buttercream Piping Masterclass
+        </h1>
+        <img class="ornament" src="./assets/white-ornament.svg"/>
+        <h2 class="hero-subtitle">
+          With pastry chef Rosario Luna
+        </h2>
+      </div>
+
+    </header>
     <!-- Signup form -->
 
+    
+
+    <form class="form" action="http://httpbin.org/anything" method="POST">
+      <h3>Please choose your date!</h3>
+
+      <br>
```

Comment:
We shouldn't use `<br />` tags, they are deprecated. Their main features is that they break the line, but you don't need it over here, `<h3 />` tag is doing that anyway. Let's delete this `<br />`